docker/scripts/functions.sh

```diff
@@ -403,23 +403,31 @@ function validate_bios_config() {
 	fi
 }
 
-# usage: bios_inventory $id $plan $facility
+# usage: bios_inventory $plan $facility
 function bios_inventory() {
-	local id=$1
-	local plan=$2
-	local facility=$3
+	local plan=$1
+	local facility=$2
 
 	# When running the inventorybios command outside of the packet-hardware
 	# container, UTIL_RACADM7 and UTIL_SUM must be set to the locations of the
 	# racadm and sum binaries, respectively
 	local bios_json_fn="/tmp/bios.json"
 	local hollow_json_fn="/tmp/hollow.json"
-	local hollow_namespace="net.equinixplatform.bios"
+	local hollow_namespace="net.platformequinix.bios"
 	if UTIL_RACADM7=/opt/dell/srvadmin/bin/idracadm7 UTIL_SUM=/opt/supermicro/sum/sum packet-hardware inventorybios --verbose -u localhost --dry --cache-file "${bios_json_fn}"; then
+		local inventorybios_json
+		inventorybios_json=$(cat ${bios_json_fn})
+		echo "inventorybios JSON is: [${inventorybios_json}]"
+
+		if [[ ${inventorybios_json} == "{}" ]]; then
+			echo "inventorybios JSON is empty, not writing to Hollow"
+			return 0
+		fi
+
 		# Generate JSON with additional fields required by Hollow
 		if ! jq --null-input \
 			--arg ns "${hollow_namespace}" \
-			--arg id "${id}" \
+			--arg id "${HARDWARE_ID}" \
```

Comment:
It feels really weird to pass this in as a global environment variable, when the other inputs are arguments. I'd recommend consistency. 

If you do need to rely on global variables passed-in from elsewhere - please document them as as a doc string in the function signature.

Reply:
The reason for this change wasn't mentioned in the commit message or PR description - I've updated both for clarity as to why. I agree that it's clearer to pass the value is a function parameter, so I've made that change as well.